src/rez/util.py

```diff
@@ -10,11 +10,35 @@
 from rez.exceptions import RezError
 from rez.utils.yaml import dump_yaml
 from rez.vendor.progress.bar import Bar
-
+from rez.vendor.enum import Enum
 
 DEV_NULL = open(os.devnull, 'w')
 
 
+class ExecutableScriptMode(Enum):
+    """
+    Which scripts to create with util.create_executable_script.
+    """
+    # Start with 1 to not collide with None checks
+
+    # Requested shebang script only. Usually extension-less.
+    requested = 1
```

Comment:
I don't really understand what is meant by "requested" here? And it isn't clear what "requested" would mean when on Windows... isn't it the same as "platform_specific"?

Would "native" be a better term? I'm trying to understand what "requested" means as in the term by the way, not the implementation!

Reply:
I renamed it to "single". Hope that makes more sense.
I left `platform_specific`, but it you prefer `native` let me know.